### PR TITLE
[commodore] Implement KERNAL LOAD function as inline assembly

### PR DESCRIPTION
**Pull request by mlund**
Using inline asm reduces the overhead of calling ROM routines significantly. The plot below shows the final size for an example C64 PRG file.
inline

### CODE REVIEW

File: mos-platform/commodore/cbm.h [hunk 1]
```diff
@@ -221,7 +221,27 @@ void cbm_k_clrch (void);
...
+ * values:
+ *
+ * `flag` | Description
+ * ------ | ---------------
+ *   0    | Load at address
+ *   1    | Verify
+ *   2    | Loads into VRAM $00000 + address
+ *   3    | Loads into VRAM $10000 + address
+ *
+ * @param flag Load (0) or verify (1-255)
```
Comment (mysterymath) [inline mos-platform/commodore/cbm.h:239]: Doesn't this conflict with the definition above?

File: mos-platform/commodore/cbm_k_load.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+// Helper to (dis)assemble a 16-bit integer
+typedef union {
+  uint16_t value;
+  struct {
+    uint8_t lo, hi;
+  };
+} Word;
+
+void *cbm_k_load(const unsigned char flag, void *load_addr) {
+
```
Comment (mysterymath) [inline mos-platform/commodore/cbm_k_load.c:17]: nit: delete blank line

Comment (mlund) [reply mos-platform/commodore/cbm_k_load.c]: fixed

File: mos-platform/commodore/cbm.h (hunk 1 above)
Comment (mlund) [reply mos-platform/commodore/cbm.h]: fixed

**Review by mysterymath — APPROVED** [review]

**Pull request merged**